Yosemite/Yosemite/Model/Extensions/URL+Helpers.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  URL+Helpers.swift
+//  Yosemite
+//
+/// Convenient extension methods for URL
+import Foundation
+
+extension URL {
+    static public func getWooCommerceAdminURL(from storesManager: StoresManager) -> URL {
+        let wooCommerceFallbackURL = URL(string: "https://woocommerce.com/blog/")
+        guard let defaultSite = storesManager.sessionManager.defaultSite else {
+            return wooCommerceFallbackURL!
+        }
+        guard let url = URL(string: defaultSite.adminURL) else {
+            if defaultSite.url.isEmpty {
```

Comment:
I wouldn't add the `wooCommerceFallbackURL` in this extension. That is strictly related to the menu view model logic, and It is not extensible to other cases where we want to get the admin URL :)

Reply:
Right! That makes sense. With the new changes I made the extension more generic 👍